stress-example/image_search/pods/annoy_indexer.yml

```diff
@@ -8,24 +8,24 @@ components:
       ref_indexer:
         !NumpyIndexer
         metas:
-          workspace: '/workspace/workspace'
+          workspace: '/docker_workspace/workspace'
           name: vec_idx
         with:
           index_filename: 'vec.gz'
     metas:
-      workspace: '/workspace/workspace'
+      workspace: '/docker_workspace/workspace'
       name: annoyidx
   - !BinaryPbIndexer
     with:
       index_filename: doc.gz
     metas:
       name: docidx
-      workspace: '/workspace/workspace'
+      workspace: '/docker_workspace/workspace'
 metas:
   name: vec_idx
   py_modules:
-    - workspace/__init__.py
-  workspace: '/workspace/workspace'
+    - docker_workspace/__init__.py
```

Comment:
this `py_modules` should not be changed

Reply:
If it's not changed it doesn't find the correct one